cl/search/models.py

```diff
@@ -2907,6 +2971,17 @@ class Meta:
         unique_together = ("citing_opinion", "cited_opinion")
 
 
+@pghistory.track(
+    pghistory.Snapshot(),
+    obj_field=None,
+)
+class OpinionOpinionsCited(OpinionsCited):
+    """A model class to track opinion opinions cited m2m relation"""
+
+    class Meta:
+        proxy = True
```

Comment:
I don't think we need this one, actually. The opinions cited from one to another are generated by eyecite, and if we need them we can always regenerate them.

Reply:
I'll remove it